e2etests/package.json

```diff
@@ -14,6 +14,12 @@
     "azure-iot-device-amqp": "1.7.4",
     "azure-iot-device-http": "1.7.4",
     "azure-iot-device-mqtt": "1.7.4",
+    "azure-iot-provisioning-service": "1.2.4",
```

Comment:
provisioning tests belong in the provisioning/e2e folder - i'd rather not mix service tests